geometry_msgs/Point[] Position_XYZ
#uint16 center_pixel_x
#uint16 center_pixel_y
uint16 upper_pixel_x
uint16 upper_pixel_y
uint16 lower_pixel_x
uint16 lower_pixel_y

uint8 counter
